lib/std/child_process.zig

```diff
@@ -257,6 +259,55 @@ pub const ChildProcess = struct {
         }
     }
 
+    fn collectOutputWindows(child: *ChildProcess, outs: [2]*std.ArrayList(u8), max_output_bytes: usize) !void {
+        const bump_amt = 512;
+        const handles = [_]windows.HANDLE{ child.stdout.?.handle, child.stderr.?.handle };
+
+        for ([_]u1{ 0, 1 }) |i| {
```

Comment:
This particular comment doesn't seem necessary for those familiar with ReadFile.  What would we have to wait for if we don't call ReadFile beforehand?

Reply:
Oh wait this doesn't actually work because `*out` makes the array list const.  It also doesn't work with `handles`.

Reply:
Don't assume people are familiar with winapi, poll works without any read() or anything like that so let's highlight the differences.

Make that work somehow or use an explicit for loop thanks.

Reply:
This is the windows asynchronous IO 101, if you've ever used it, then you know how this works.  That being said, it doesn't hurt to explain it here since it's very different from the posix world, so at your request I've added an educational comment here.

Reply:
Again, don't expect the reader to be a winapi master. If shit breaks and someone tries to fix it I want them to find some useful pointers to get started.

Reply:
Again, it's 101, not "master" level.

Reply:
All this fuss over a single-line comment? Talk about wasting people's time...

Reply:
I really don't appreciate this passive aggressive behavior. You're both respected members of the community and we have plenty of young students who are looking up to you both, taking subconscious notes on how to treat their peers. We also have lurkers whose participation may depend on how friendly they perceive the code review process to be.

How you two treat each other is far more important than the code quality at the end of any given review process. We don't need any contributors getting burnt out from bummer interactions.

Reply:
For those reading this after the fact, I made some harsh comments here that I shouldn't have.  But I will leave there here as an example of what not to say and I'll do better.

Reply:
It was just some cheeky banter, I agree the peanut gallery may not perceive this exchange as such so sorry about that. No feelings were hurt (and hopefully I didn't hurt any feelings).
Now, back to technical matters.

Reply:
That's good to hear, I'm glad you two didn't develop some kind of weird grudge against each other :)